community/security/src/test/java/org/neo4j/server/security/auth/FileUserRepositoryTest.java

```diff
@@ -77,7 +83,7 @@ public void shouldStoreAndRetriveUsersByName() throws Exception
     {
         // Given
         FileUserRepository users = new FileUserRepository( authFile, NullLogProvider.getInstance() );
-        User user = new User( "jake", Credential.INACCESSIBLE, true );
+        User user = new User( "jake", "admin", Credential.INACCESSIBLE, true );
```

Comment:
We are not testing the group. The reason group is in community is for backwards compatibility of the auth file and to simplify the architecture. The group is ignored and has no meaning to the community auth manager.